packages/opentelemetry-propagator-jaeger/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@opentelemetry/propagator-jaeger",
+  "version": "0.3.2",
+  "description": "OpenTelemetry propagator Jaeger provide HTTP header propagation for systems that using Jaeger HTTP header format",
```

Comment:
And the same here?:
`OpenTelemetry Jaeger Propagator provides HTTP header propagation for systems that are using Jaeger HTTP header format.`